scalyr_agent/scalyr_client.py

```diff
@@ -565,7 +565,9 @@ def generate_body():
             return add_events_request.get_payload()
 
         return self.__send_request(
-            "/addEvents", body_func=generate_body, block_on_response=block_on_response
+            six.ensure_str("/addEvents"),
```

Comment:
`Httplib`(if used)  builds request just by concatenation of request path, body, end headers.
When request path or headers are `unicode`, it tries to decode body implicitly before concatenation, but it can't because it has no utf-8 data. That's why I use `six.ensure_str`, as it's name implies, it keeps 'str' on both versions, e.g. binary in python2 and text in python3